packages/internals/src/get-generators/generatorResolvers/prisma-client-js/check-dependencies/resolve.ts

```diff
@@ -13,6 +21,14 @@ async function resolve(id: string, options: _resolve.AsyncOpts) {
   }) as Promise<string | undefined>
 }
 
+/**
+ * Alternative to resolve that returns undefined instead of throwing. It also
+ * enables preserving symlinks, which is not possible with the original resolve.
+ * This variant will find the root of a package.
```

Comment:
```suggestion
 * Alternative to 'resolve' that returns undefined instead of throwing. It also
 * enables preserving symlinks, which is not possible with the original 'resolve'.
 * This variant will find the _root_ of a package.
```